decidim-core/db/seeds.rb

```diff
@@ -117,6 +118,7 @@
     tos_agreement: true,
     personal_url: Faker::Internet.url,
     about: Faker::Lorem.paragraph(2)
+    # accepted_tos_version: Time.current
```

Comment:
🧐

Reply:
same as before :crying_cat_face: I'll fit it